### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by megheaiulian**
Add missing tests. Some of them will fail.

### CODE REVIEW

File: test/cosmoz-datehelper-behavior-test.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+<head>
+  <title>cosmoz-datehelper-behavior test</title>
+  <meta charset='utf-8'>
+  <meta name='viewport' content='width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes'>
+  <script src='../../webcomponentsjs/webcomponents-lite.js'></script>
+  <script src='../../web-component-tester/browser.js'></script>
+  <link rel='import' href='./fixtures/cosmoz-datehelper-behavior.html'>
+</head>
+<body>s
```
Comment (nomego) [inline test/cosmoz-datehelper-behavior-test.html]: S slipped in

File: test/cosmoz-datehelper-behavior-test.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	<test-fixture id="cosmoz-datehelper-behavior-fixture">
+		<template>
+			<test-cosmoz-datehelper-behavior></test-cosmoz-datehelper-behavior>
+		</template>
+	</test-fixture>
+  <script type="text/javascript">
+  (function () {
+		'use strict';
```
Comment (nomego) [inline test/cosmoz-datehelper-behavior-test.html]: Different indents

File: test/cosmoz-datehelper-behavior-test.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    
+    suite('cosmoz-datehelper-behavior' ,function() {
```
Comment (nomego) [inline test/cosmoz-datehelper-behavior-test.html]: Missing spacing, different multi var declaration style

File: test/cosmoz-datehelper-behavior-test.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      var dateHelper
+        , date = new Date("October 13, 2014 11:13:20")
+        , futureDate = new Date("October 13, 2022 11:13:20")
+        , invalidDate = ''
+        , undefinedDate
+        , dateNow, dateHourAgo, dateWeekDiff = new Date(date.getTime() - 1000 * 60 * 60 * 24 * 7) ;
+      
+      
+      setup(function() {
```
Comment (nomego) [inline test/cosmoz-datehelper-behavior-test.html]: Missing spacing between function and parenthesis on all anon funcs

File: cosmoz-datehelper-behavior.js [hunk 5]
```diff
@@ -71,10 +58,14 @@
 			}
 			return ensuredDate;
 		},
+
 		dayDiff: function (date1, date2) {
-			var ensuredDate1 = this.ensureDate(date1)
-				, ensuredDate2 = this.ensureDate(date2);
-			return Math.round((ensuredDate1 - ensuredDate2) / (1000 * 60 * 60 * 24));
+			var time1 = moment(date1)
+			, time2 = moment(date2);
```
Comment (nomego) [inline cosmoz-datehelper-behavior.js]: multi-var-declaration-style

File: test/cosmoz-datehelper-behavior-test.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+			<test-cosmoz-datehelper-behavior></test-cosmoz-datehelper-behavior>
+		</template>
+	</test-fixture>
+	<script type="text/javascript">
+	(function () {
+		'use strict';
+
+		suite('cosmoz-datehelper-behavior' , function () {
+			var dateHelper
+				, date = new Date('October 13, 2014 11:13:20')
```
Comment (nomego) [inline test/cosmoz-datehelper-behavior-test.html]: multi-var-declaration-style

File: test/cosmoz-moneyhelper-behavior-test.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+			var moneyHelper,
+				amount = 2200200.3333333333;
+
+
+			setup(function () {
+				moneyHelper = fixture('cosmoz-moneyhelper-behavior-fixture');
+			});
+
+			test('faketest ', function (done) {
+				assert.equal(moneyHelper.renderAmount({amount: amount}),'$2,200,200.33');
```
Comment (nomego) [inline test/cosmoz-moneyhelper-behavior-test.html]: shouldn't USD be specified ?

Comment (megheaiulian) [inline test/cosmoz-moneyhelper-behavior-test.html]: It's kind of testing that USD is default also.

**Pull request merged**